operator/v1/types_network.go

```diff
@@ -96,12 +96,32 @@ type DefaultNetworkDefinition struct {
 	OVNKubernetesConfig *OVNKubernetesConfig `json:"ovnKubernetesConfig,omitempty"`
 }
 
-// AdditionalNetworkDefinition configures an extra network that is available but not
+// MacVlanConfig contains configurations for macvlan interface.
+type MacVlanConfig struct {
+
+	// Host interface to enslave interface.
+	Master string `json:"master"`
+
+	// Ipam indicates which IPAM module will be used for IP Address Management(IPAM)
+	// Ipam for now supports 'dhcp'
+	// +optional
+	IPAM string `json:"ipam,omitempty"`
+
+	// Mode is the macvlan mode: bridge, private, vepa, passthru. The default is bridge
+	// +optional
+	Mode string `json:"mode,omitempty"`
+
+	// MTU is the mtu to use for the tunnel interface. Defaults to the value from kernel if unset.
```

Comment:
"for the interface". It's not a tunnel. Also, it's not clear why this should be needed for macvlan. Did you just copy this from `OpenShiftSDNConfig`?

Reply:
MTU is required especially for tuning of network traffic. In some case, user want to change macvlan interface from master interface to optimize traffic (especially TCP). So I'd like to keep it.

Regarding the comment, I've fixed.